getting-started/kubernetes/k3s/quickstart.md

```diff
@@ -39,11 +39,44 @@ development.
    > CIDR by replacing 192.168.0.0/16 in the above command. 
    {: .alert .alert-danger}
 
+   K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`
 
+   ```
+   sudo su
+   ```
+
+### Install {{site.prodname}}
+
+   > **Note**: As of v3.15 milestone, manifest installation will gradually be replaced by {% include open-new-window.html text='Tigera operator' url='https://github.com/tigera/operator' %}.
+   > Tigera operator is another opensource solution from Tigera team which will deliver a better experience when you install or upgrade your
```

Comment:
Tigera operator is another open source solution from Tigera that delivers a better experience when you install or upgrade your